it/controllers/benefits/AccommodationRelocationControllerISpec.scala

```diff
@@ -0,0 +1,364 @@
+          welshToggleCheck(user.isWelsh)
+
+        }
+
+        "Render the Accommodation Relocation Benefits Question page with 'yes' as pre-filled data" which {
+          lazy val result: WSResponse = {
+            authoriseAgentOrIndividual(user.isAgent)
+            dropEmploymentDB()
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("name", hmrc = true, Some(benefits(Some(filledAccModel(true)))))), userRequest)
+            userDataStub(userData(fullEmploymentsModel(hmrcEmployment = Seq(employmentDetailsAndBenefits(fullBenefits)))), nino, taxYearEOY)
+            urlGet(url(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          import Selectors._
+          import user.commonExpectedResults._
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedH1)
+          captionCheck(user.commonExpectedResults.expectedCaption(taxYearEOY))
+          radioButtonCheck(yesText, 1, Some(true))
+          radioButtonCheck(noText, 2, Some(false))
+          buttonCheck(expectedButtonText, continueButtonSelector)
+          formPostLinkCheck(continueLink, continueButtonFormSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+
+        "Redirect to Check Employment Benefits page" when {
+          "there is no in-session data for the user" which {
+            lazy val result: WSResponse = {
+              authoriseAgentOrIndividual(user.isAgent)
+              dropEmploymentDB()
+              urlGet(url(taxYearEOY), welsh = user.isWelsh, follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+            }
+            s"has an SEE_OTHER($SEE_OTHER) status" in {
+              result.status shouldBe SEE_OTHER
+              result.header("location") shouldBe Some(CheckYourBenefitsController.show(taxYearEOY, employmentId).url)
+            }
+          }
+
+          "there is in-session data but user has no benefits" which {
+            lazy val result: WSResponse = {
+              authoriseAgentOrIndividual(user.isAgent)
+              dropEmploymentDB()
+              insertCyaData(employmentUserData(isPrior = true, cyaModel("name", hmrc = true)), userRequest)
+              urlGet(url(taxYearEOY), welsh = user.isWelsh, follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+            }
+            s"has an SEE_OTHER($SEE_OTHER) status" in {
+              result.status shouldBe SEE_OTHER
+              result.header("location") shouldBe Some(CheckYourBenefitsController.show(taxYearEOY, employmentId).url)
+            }
+          }
+
+          "it is not EOY" which {
```

Comment:
the "not EOY" test can be moved so it's not inside of the brackets for "Redirect to Check Employment Benefits page"